tests/test_apis.py

```diff
@@ -942,6 +942,18 @@ def testAppend(self):
         self.assertEqual('|' not in md2.ESCAPED_CHARS, True)
 
 
+class TestBlockAppend(unittest.TestCase):
```

Comment:
@waylan , wasn't sure if you have a special place for **new** API tests, so I put this with the old.

Reply:
This isn't actually testing rendered output, so this is were it belongs.